### PR TITLE
Use engine-p style in any

### PR DESCRIPTION
**Pull request by LhKipp**

### CODE REVIEW

File: crates/nu-command/src/commands/any.rs [hunk 1]
```diff
@@ -52,74 +50,88 @@ impl WholeStreamCommand for Command {
...
-        UntaggedValue::boolean(false).into_value(&tag),
-    ));
+    let scope = args.scope.clone();
+
+    // Variables in nu are immutable. Having the same variable accross invocations
+    // of evaluate_baseline_expr does not mutate the variables and thus each
+    // invocations are independent of each other!
+    scope.enter_scope();
+    scope.add_vars(&any_args.predicate.captured.entries);
+    let result: Result<bool, ShellError> = args.input.try_any(|row| {
```
Comment (jonathandturner) [inline crates/nu-command/src/commands/any.rs:91]: I'm wondering what the advantage of `.try_any` is here over the built-in `.any`?

Comment (LhKipp) [reply crates/nu-command/src/commands/any.rs:91]: try_any allows for short_circuiting here. (like the other try_x commands in std::iter::Iterator)

Comment (jonathandturner) [reply crates/nu-command/src/commands/any.rs:91]: Yeah, I did figure it out once I saw your updates to the other PR. Some comments here: https://github.com/nushell/nushell/pull/3312#discussion_r615295399

File: crates/nu-command/src/commands/any.rs [hunk 2]
```diff
@@ -52,74 +50,89 @@ impl WholeStreamCommand for Command {
...
-                },
-                Err(e) => Err(e),
+        condition.as_bool()
+    });
+    scope.exit_scope();
+
+    Ok(OutputStream::one(result?))
+}
+
+trait TryAnyExt: Iterator {
```
Comment (jonathandturner) [inline crates/nu-command/src/commands/any.rs:105]: Just like the other PR, can you remove the extension here as well?

**Comment by jonathandturner** [comment]
Looks good

**Pull request merged**